bindata/kube-proxy/001-rbac.yaml

```diff
@@ -21,6 +21,13 @@ rules:
   - get
   - list
   - watch
+- apiGroups: ["authentication.k8s.io"]
+  resources:
+  - tokenreviews
```

Comment:
never used this resource, sorry
is kube-proxy using it?

Reply:
Yes, apparently: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/15053/rehearse-15053-release-openshift-origin-installer-e2e-aws-calico-4.7/1351588780695359488/artifacts/e2e-aws/pods/openshift-kube-proxy_openshift-kube-proxy-56vl7_kube-rbac-proxy.log

Anyways, I see openshift-sdn has the same rbac and it seems linked to metrics collection, @juanluisvaladas do you know if only `authentication.k8s.io/tokenreviews` is needed or if I should also add `authorization.k8s.io/subjectaccessreviews` as you did in your PR: https://github.com/openshift/cluster-network-operator/pull/574

Reply:
Actually, I noticed all metrics related PRs seem to apply the same RBAC permissions, so I am doing it here too.